### DISCUSSION TITLE
Robustness and strict=False documentation

### DISCUSSION THREAD

**Discussion by j-t-1** [py-pdf/pypdf#2478] [Q&A] [answered]
pypdf has three core objects and all of them have a strict parameter:

• PdfReader
• PdfWriter
• PdfMerger
https://pypdf.readthedocs.io/en/latest/user/robustness.html

However, in the PdfWriter documentation, there is not a strict parameter.
https://pypdf.readthedocs.io/en/latest/modules/PdfWriter.html

The PdfMerger documentation, says it is deprecated since version 5.0.0.
https://pypdf.readthedocs.io/en/latest/modules/PdfMerger.html

So does this mean that robustness.html needs amending?

pypdf has two core objects:

• PdfReader
• PdfWriter

Only the PdfReader has a strict parameter, since presumably you do not want to
write a non-conforming pdf.

**Comment by stefan6419846** [comment, marked as answer]
Thanks for reporting this inconsistency. Do you want to submit a corresponding PR?

**Reply by j-t-1** [reply]
If the amendment suggestion is correct, sure.

Where is the file for this in the repo?

**Reply by stefan6419846** [reply]
https://github.com/py-pdf/pypdf/blob/main/docs/user/robustness.md
